runtime/commonTest/src/kotlinx/serialization/TuplesTest.kt

```diff
@@ -22,16 +23,19 @@ class TuplesTest {
     fun testCustomPair() = assertStringFormAndRestored(
         "{k:42,v:foo}",
         MyPair(42, "foo"),
-        MyPair.serializer(IntSerializer, StringSerializer),
-        Json.unquoted
+        MyPair.serializer(
+            Int.serializer(),
+            String.serializer()
+        ),
+        Json { unquoted = true }
     )
 
     @Test
     fun testStandardPair() = assertStringFormAndRestored(
         "{p:{first:42,second:foo}}",
         PairWrapper(42 to "foo"),
         PairWrapper.serializer(),
-        Json.unquoted
+        Json { unquoted = true }
```

Comment:
Since unquoted is widely used in our tests, you can do the following:

```kotlin
private val testUnquoted = Json { unquoted = true }

val Json.Default.unquoted: Json get() = testUnquoted
```

Btw, `Default` as a name for companion looks really ugly in this context. Maybe we don't have to rename them?

Reply:
`Default` is named so for the same reasons it is done in `kotlin.Random`:
  -- Documentation is *much* prettier on the site
  --  Stacktraces are better